api/utils/keys/yubikey.go

```diff
@@ -254,8 +252,12 @@ func (y *YubiKeyPrivateKey) Sign(rand io.Reader, digest []byte, opts crypto.Sign
 	y.signMux.Lock()
 	defer y.signMux.Unlock()
 
+	// For generic auth errors, the smart card returns the error code 0x6982. This PIV library
```

Comment:
nit: 0x6982 is not the same as decimal 6982, is that correct?

Reply:
Yes, The PIV library deals with it as a hexadecimal `0x6982` but ultimately stringifies it as "6982" in the error message.

Reply:
I think that's worth calling out in the comment, as it seems like an error liable to be fixed by them (probably by printing 0x6982, I'd imagine, but it could be translated to decimal too).

Reply:
They're doing `fmt.Sprintf("smart card error %04x%s", 0x6982, "security status not satisfied")`, so it looks intentional.